notion/manifest.json

```diff
@@ -0,0 +1,43 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "bb61af26-0004-11ee-81a2-da7ad0900002",
+  "app_id": "notion",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Monitor your Notion workspace events and customize detections in Datadog",
+    "title": "Notion",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Setting up the Datadog integration in Notion",
+        "image_url": "images/Notion_DD.png"
+      },
+      {
+        "media_type": "image",
+        "caption": "Notion event logs in Datadog",
+        "image_url": "images/Notion_DD_2.png"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Event Management",
+      "Category::Collaboration",
+      "Supported OS::Any"
+    ]
```

Comment:
Please included relevant submitted query data from the list [here](https://docs.datadoghq.com/developers/integrations/check_references/#manifest-file). You can ctrl+f for "query data". Thanks!

Reply:
@emarsha94, did a search for "query data" and got 0 results. I think I've already added all relevant classifier tags for what we're doing

Reply:
Supported OS should match the formatting as seen in [this document](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags). Please update the formatting 

As for Query data, this is required to add if you are pulling out data from Datadog. If you are sending in data, please add the Submitted Data Type as seen in the doc linked above. If you are pulling out data, it will be Queried Data Type

Reply:
Is my formatting not correct? I was referring to this https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags:~:text=data%20from%20Android.-,Supported%20OS%3A%3AAny,-An%20integration%20that

To clarify Notion has a web and native (Windows, Mac, iOS, Android) app. Is the supported OS in terms of directly from the OS logging, if so, I can remove it.